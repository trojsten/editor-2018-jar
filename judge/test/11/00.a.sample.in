STR:
General
Zaba
STR:
Plukovnik
Vlejd
STR:
Podplukovnik
Baska
STR:
Strateg
Matus
STR:
Spion
Ralbo
STR:
Programator
Hodobox
STR:
Velmajstermaxisupergeneralveduci
Misof
STR:
Nadpodpredzadplukovnikazarovendokoncaajgeneral
Korman
