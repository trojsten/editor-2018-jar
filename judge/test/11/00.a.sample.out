{"Velmajstermaxisupergeneralveduci": "Korman", "Spion": "Hodobox", "Nadpodpredzadplukovnikazarovendokoncaajgeneral": "Misof", "Programator": "Vlejd", "General": "Matus", "Strateg": "Ralbo", "Podplukovnik": "Zaba", "Plukovnik": "Baska"}